cloudant-http/src/main/java/com/cloudant/http/internal/interceptors/CookieInterceptor.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2015, 2016 IBM Corp. All rights reserved.
+ * Copyright © 2017 IBM Corp. All rights reserved.
```

Comment:
Copyrights should be in the form `initial-year, current-year` or `2015, 2017` for this file